### ISSUE TITLE
Crash on some device AndroidPlayInstallReferrerPlugin.java line 51

### ISSUE THREAD

**Issue by slorop41314** [lschmierer/android_play_install_referrer#14]
Package used:
  android_play_install_referrer: ^0.1.1
  

<img width="992" alt="Screen Shot 2021-07-27 at 10 51 42" src="https://user-images.githubusercontent.com/52713279/127092609-71283a1a-0f3b-426c-af48-b84f3054ae81.png">

**Comment by stwonary** [comment]
same here but on line 46

```
de.lschmierer.android_play_install_referrer.AndroidPlayInstallReferrerPlugin$getInstallReferrer$1.onInstallReferrerSetupFinished (AndroidPlayInstallReferrerPlugin.java:46) 
```

**Comment by lschmierer** [comment]
Was someone able to reliable reproduce this issue?
If so, can you check if https://github.com/lschmierer/android_play_install_referrer/pull/17 resolves this issue?

**Comment by lschmierer** [comment]
I have published a possible fix as 0.2.0 

Closing this for now.

**Comment by stwonary** [comment]
we're unable to reproduce this issue. will update to 0.2.0 and release it to production. 

will report back if the issue persist. thank you @lschmierer
